torch/optim/swa_utils.py

```diff
@@ -43,6 +46,8 @@ def ema_update(ema_param_list: PARAM_LIST, current_param_list: PARAM_LIST, _):
 
 
 def get_swa_multi_avg_fn():
+    """Get the function applying stochastic weight average (SWA) across a multiple params."""
```

Comment:
```suggestion
    """Get the function applying stochastic weight average (SWA) across multiple params."""
```

Reply:
err, my bad sorry! Thanks for catching @janeyx99

Reply:
Addressed your comment @janeyx99 Thanks!